resources/Makefile.j2

```diff
@@ -0,0 +1,111 @@
+MAKEFLAGS 				+= --warn-undefined-variables
+SHELL 					:= bash
+.DEFAULT_GOAL			:= help
+URIBASE					:=	http://purl.obolibrary.org/obo
+TEST_DATA				:=	testdata
+TMP_DATA				:=	{{ directories.tmp }}
+NAME					:= $(shell python -c 'import tomli; print(tomli.load(open("pyproject.toml", "rb"))["tool"]["poetry"]["name"])')
+VERSION					:= $(shell python -c 'import tomli; print(tomli.load(open("pyproject.toml", "rb"))["tool"]["poetry"]["version"])')
+H2_JAR					:= {{ directories.h2jar }}
+help: status
+	@echo ""
+	@echo "help"
+	@echo "make pheval -- this runs the entire pipeline including corpus preparation and pheval run"
+	@echo "make semsim -- generate all configured similarity profiles"
+	@echo "make semsim-shuffle -- generate new ontology terms to the semsim process"
+	@echo "make semsim-scramble -- scramble semsim profile"
+	@echo "make semsim-convert -- convert all semsim profiles into exomiser SQL format"
+	@echo "make semsim-ingest -- takes all the configured semsim profiles and loads them into the exomiser databases"
+
+	@echo "make clean -- removes corpora and pheval results"
+	@echo "make help -- show this help"
+	@echo ""
+
+status:
+	@echo "Project: $(NAME)"
+	@echo "Version: $(VERSION)"
+
+.SECONDEXPANSION:
+
+
+
+.PHONY: prepare-inputs
+
+{% for config in configs %}
+
+
+prepare-inputs += configurations/{{ config.tool }}-{{ config.version }}-{{ config.configuration }}/config.yaml
+
+
+configurations/{{ config.tool }}-{{ config.version }}-{{ config.configuration }}/config.yaml:
+	mkdir -p $(shell dirname pwd)/$(shell dirname $@)
+	ln -s {{ directories[config.tool] }}/* $(shell dirname pwd)/$(shell dirname $@)
+
+
+{% if config.tool == 'exomiser' %}
+configurations/{{ config.tool }}-{{ config.version }}/{{ config.configuration }}/{{ phenotype.version }}_phenotype/{{ phenotype.version }}_phenotype.h2.db: configurations/{{ config.tool }}-{{ config.version }}-{{ config.configuration }}/config.yaml configurations/{{ config.configuration }}/hp-mp.semsim.{{ config.configuration }}.sql configurations/{{ config.configuration }}/hp-mp2.semsim.{{ config.configuration }}.sql
```

Comment:
`hp-mp` needs to be parametrised in the config, maybe with an option `semantic_similarity_profile`?

The word hp-mp should not appear in this makefile at all